src/main/java/mate/academy/springbootstore/controller/BookController.java

```diff
@@ -1,4 +1,39 @@
 package mate.academy.springbootstore.controller;
 
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import mate.academy.springbootstore.dto.BookDto;
+import mate.academy.springbootstore.dto.CreateBookRequestDto;
+import mate.academy.springbootstore.service.BookService;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/api/books") // виправлено

```

Comment:
```suggestion
@RequestMapping("/api/books")
```
You can use `server.servlet.context-path` for the /api prefix